### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by linwaytin** [sswatson/AsyPlots.jl#5]
Hi, is this project still maintained?
Recently `Asymptote` can output html file using WebGL, which can show nice 3D graphics in modern browsers.
I'm curious if this feature can be  added to `AsyPlots`, since old 3D functions seem not working.

**Comment by sswatson** [comment]
I still use it, yes. I just pushed an update if you want to check out master and see if the 3D graphs work for you. It works fine on my machine.

I noticed the news about Asymptote's support for WebGL as well. You can use this package for that. It can output an asy file instead of a PDF (just use the .asy extension when you save the file). You can then obtain a WebGL 3D version from the asy file in the usual way.

**Comment by linwaytin** [comment]
Great!
Both `Asymptote` and `Julia` are great tools. I'm glad to see you make a bridge between them!

**Comment by sswatson** [comment]
Yes, I'm very glad to have it available. I was into it more when I first wrote it, but these days I'm focusing more on tools that emphasize interactivity (like [JSXGraph](https://jsxgraph.uni-bayreuth.de/wp/index.html)), which Asymptote is not really built for. Still, there's nothing I like as much Asymptote for really sharp looking static figures (particularly 3D!).

**Comment by linwaytin** [comment]
`JSXGraph` looks cool. I don't even know this library. Thanks!
I agree that `Asymptote` is not for interactivity. It's built for the quality of static figures.
I think its 3D picture quality is amazing, maybe close to commercial software.
For interactivity, with `Pluto` we can achieve it to some extent. 
I'm curious how well `AsyPlots` is integrated with `Pluto`, which is a really nice notebook.
If I could, I would like to make `Asymptote` integrated more with `Julia`.
It's really a nice tool for technical plotting, but somehow not very popular. 
`TikZ` is much more popular, but I really don't like it. 
IMO, the only thing `TikZ` wins is its integration with LaTeX.
So I really appreciate you made this package.
